packages/create-vite/src/index.ts

```diff
@@ -286,6 +286,25 @@ const FRAMEWORKS: Framework[] = [
       },
     ],
   },
+  {
+    name: 'ember',
+    display: 'Ember',
+    color: redBright,
+    variants: [
+      {
+        name: 'ember-app-ts',
+        display: 'TypeScript ↗',
+        color: blueBright,
+        customCommand: 'npx ember-cli@latest new TARGET_DIR --typescript',
+      },
+      {
+        name: 'ember-app',
+        display: 'JavaScript ↗',
+        color: redBright,
+        customCommand: 'npx ember-cli@latest new TARGET_DIR',
```

Comment:
```suggestion
        customCommand: 'npm exec -- ember-cli@latest new TARGET_DIR --typescript',
      },
      {
        name: 'ember-app',
        display: 'JavaScript ↗',
        color: redBright,
        customCommand: 'npm exec -- ember-cli@latest new TARGET_DIR',
```
Just noticed that we should use `npm exec` instead of `npx` here because of https://github.com/vitejs/vite/blob/18fb620adcc88906848ab56528ad6a72bedd51e2/packages/create-vite/src/index.ts#L874